jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -423,6 +425,9 @@ def mos_niriss_parser(app, data_dir, obs_label=""):
     image_dict = {}
     filter_wcs = {}
 
+    # Set up a dictionary of datasets to add to glue
+    add_to_glue = OrderedDict()
```

Comment:
Why not just use `dict`? It is ordered now, right?

Reply:
Yes.

https://github.com/spacetelescope/jdaviz/blob/8019780dcb4d9198b2bb4b1b5a0d7392d4508969/setup.cfg#L18

Reply:
Done